docs/src/blockchain-development/native_assets.md

```diff
@@ -7,3 +7,23 @@ What does this mean in practice?
 As in Ethereum, sending ETH to an address or contract is an operation built into the FuelVM, meaning it doesn't rely on the existence of some token smart contract to update balances to track ownership.
 
 However, unlike Ethereum, the process for sending _any_ native asset is the same. This means that while you would still need a smart contract to handle the minting and burning of fungible tokens, the sending and receiving of these tokens can be done independently of the token contract.
+
+## Liquidity Pool Example
+
+All contracts in Fuel can mint and burn their own native token. Contracts also manage an internal balance of all native assets pushed through calls or minted by the contract. Lastly, contracts can receive and transfer any native asset including their own.
```

Comment:
Doesn't the protocol keep track of the balances of native assets?
The contract shouldn't need to do any accounting of it's own native assets.

Reply:
That is, you can always query any contract's balance of any native asset.

Reply:
Oh that's what I'm trying to refer to here. That the contract by virtue of the VM is tracking a balance.

Perhaps there is a clearer way to write that. Suggestions welcome.

Reply:
I think that's better, with a small tweak:
"Therefore, there is no need for any manual accounting of the contract's balances using persistent storage."